src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -115,6 +146,9 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
           '-'
         )}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.triggeredBy}>
+        {getTriggerredByColumnData()}
```

Comment:
You can simply render `{icon <ExternalLink/>}` here and do all the computation in the component